PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -33,6 +33,7 @@
 Flag_muonBadTrackFilter = cms.Path(muonBadTrackFilter)
 Flag_BadChargedCandidateFilter = cms.Path(BadChargedCandidateFilter)
 Flag_BadPFMuonFilter = cms.Path(BadPFMuonFilter)
+Flag_BadPFMuonDzFilter  = cms.Path(BadPFMuonDzFilter) 
 Flag_BadChargedCandidateSummer16Filter = cms.Path(BadChargedCandidateSummer16Filter)
 Flag_BadPFMuonSummer16Filter = cms.Path(BadPFMuonSummer16Filter)
```

Comment:
this does not match the master version, even though the update is more self-consistent.
Still, where possible, the backports should be identical to the original/master version.

In the master version 
https://github.com/cms-sw/cmssw/blob/a37e1d9101e88827f6960623c8549894f3601cd4/PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py#L35-L38

Reply:
done